File: ml-data.c
#include <stdio.h>
#include "ninstances-c/ninstances.h"

int main(void)
{
int x, flag;
flag=1;

while(flag==1)
{
printf("Menu:\n");
printf("0.- EXIT\n");
printf("1.- Number of instances, features and labels (Juan A. Romero)\n");
scanf("%d", &x);
fflush(stdin);
printf("\n\n");
switch(x){
    case 0:
        flag=0;
        break;
    case 1:
        printf("Ejecutando la función ninstances() de Juan A. Romero:\n");
        ninstances("yeast.complete");
        break;
    case 2:
        printf("Funcionalidad de prueba añadida por juanrome@gmail.com");
        break;
    case 3:
        printf("NUeva Funcionalidad AAABBBBBBB");
/* add here more functionality */
    }
}

}

File: ml-data.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
from ninstances_py.ninstances import ninstances

def main():
    loop_condition=True

    while loop_condition == True:
        print("\nMenu:\n")
        print("0.- EXIT")
        print("1.- Number of instances, features and labels (Juan A. Romero)")

        main_input = int(input())

        if main_input == 0:
            loop_condition = False
            break
        elif main_input == 1:
            datafile=input("Introduce nombre de datafile: ")
            ninstances(datafile)
        elif main_input == 2:
            print("add yours here")


if __name__ == "__main__":
    main()
